Makefile

```diff
@@ -1,7 +1,7 @@
 # Copyright 2018-2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-.DEFAULT_GOAL := release
+.DEFAULT_GOAL := all
```

Comment:
@rtjohnso - **NOTE**: All these diffs in `Makefile` show up as I rebased v/s /main. Seems like this reapplied the diffs you had introduced, too. (Not sure how hard it will make your effort to cherry-pick this to your branch.)